FEXCore/Source/Interface/Core/JIT/Arm64/MemoryOps.cpp

```diff
@@ -2049,11 +2103,53 @@ DEF_OP(MemCpy) {
     const int32_t OpSize = Size;
     const int32_t SizeDirection = Size * Direction;
 
-    ARMEmitter::BackwardLabel AgainInternal{};
-    ARMEmitter::SingleUseForwardLabel DoneInternal{};
+    ARMEmitter::BiDirectionalLabel AgainInternal{};
+    ARMEmitter::ForwardLabel DoneInternal{};
 
     // Early exit if zero count.
     cbz(ARMEmitter::Size::i64Bit, TMP1, &DoneInternal);
+    orr(ARMEmitter::Size::i64Bit, TMP4, TMP2, TMP3);
```

Comment:
Should this be in the if, to skip it in the IsAtomic case?

Reply:
ah oops, fixed